.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
 Layout/LineLength:
+  Max: 100
+  Exclude:
+    - "spec/**/*_spec.rb"
+Layout/SpaceInLambdaLiteral:
   Enabled: false
-Lint/StructNewOverride:
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
+  EnforcedStyle: no_space
+  EnforcedStyleForEmptyBraces: no_space
+Lint/BooleanSymbol:
+  Enabled: false
+Lint/StructNewOverride:
+  Enabled: false
 Lint/RaiseException:
+  Enabled: false
+Lint/SuppressedException:
+  Exclude:
+    - "spec/spec_helper.rb"
+Metrics/AbcSize:
+  Max: 25
+Metrics/CyclomaticComplexity:
   Enabled: true
+  Max: 12
+Metrics/MethodLength:
+  Enabled: false
+Metrics/ClassLength:
+  Enabled: false
 Metrics/BlockLength:
-  Exclude:
-    - "spec/**/*"
-    - "tmp/**/*"
+  Enabled: false
 Naming/FileName:
   Enabled: false
+# Naming/FileName:
+#   Exclude:
+#     - "lib/*-*.rb"
+Naming/MemoizedInstanceVariableName:
+  Enabled: false
+Naming/MethodName:
+  Enabled: false
+Naming/PredicateName:
+  Enabled: false
 Naming/RescuedExceptionsVariableName:
   PreferredName: exception
+Style/AccessModifierDeclarations:
+  Enabled: false
```

Comment:
@timriley Isn't this a good way to avoid mixed styles: grouped `private` vs inline `private`?

Reply:
Seems like this was added to the dry-rb rules over a year ago. I'm not sure why it came in (if this is a common theme, it's because I only started looking at dry-rb's rules, and suggesting a few changes, relatively recently).

I'm willing to concede that there may be a case to make for mixed styles in certain circumstances, but I think we could remove this from Hanami.